index.js

```diff
@@ -67,7 +95,18 @@ function getAverageIMDBRating() {}
  *  countByRating(movies);
  *  //> { G: 3, PG: 7 }
  */
-function countByRating() {}
+function countByRating(movies) {
+  let newObj = {};
+  for (let i = 0; i < movies.length; i++){
+    if(!newObj[movies[i].rated]){
+      newObj[movies[i].rated] = 1;
+    }else {
+      newObj[movies[i].rated] += 1;
+    }
+  }
+  return newObj;
+
+}
```

Comment:
This code is gorgeous. just watch the spacing on the `else`, after the for loop, and the last line.